src/wrappers/PhylogenyWrapper.js

```diff
@@ -431,7 +431,7 @@ class PhylogenyWrapper {
     // Set name and class for phylogeny. If no '@id' is set, use the provided
     // fallbackIRI.
     if (!has(phylogenyAsJSONLD, '@id')) phylogenyAsJSONLD['@id'] = fallbackIRI;
-    phylogenyAsJSONLD['@type'] = owlterms.PHYLOREFERENCE_PHYLOGENY;
+    phylogenyAsJSONLD['@type'] = owlterms.CDAO_ROOTED_TREE;
```

Comment:
How about http://ontology.phyloref.org/phyloref.owl#ReferencePhylogenyEvidence. Wouldn't that be more appropriate? It's not currently asserted as a subclass of CDAO:rooted_tree, but that could be added.

Reply:
Yes, that's better! I've made that change in 640c4cb4bf09.